addons/test_mail/tests/test_mail_channel.py

```diff
@@ -238,12 +245,14 @@ def test_channel_auto_unsubscribe_archived_or_deleted_users(self):
         self.assertEqual(test_channel_private.channel_partner_ids, test_partner)
         self.assertEqual(test_channel_group.channel_partner_ids, test_partner)
         self.assertEqual(self.test_channel.channel_partner_ids, self.user_employee.partner_id | test_partner)
+        self.assertEqual(test_chat.channel_partner_ids, self.user_employee.partner_id | test_partner)
```

Comment:
adapt the comment just above "but not from public channels" -> "and not from chat"
same adaptation for the delete case

Reply:
changes are done.